CONTRIBUTING.md

```diff
@@ -1,146 +1,143 @@
+   - All PR's are reviewed by the Maintainers and Eng/Dev first, before being approved and committed for Eng/development.
 
-Our maintainers are:
 
-- [@gila][members-gila] - [@mayadata-io][maya-data]
-- [@jkryl][members-jkryl] - [@mayadata-io][maya-data]
-- [@GlennBullingham][members-glennbullingham] - [@mayadata-io][maya-data]
+> The processes above are documented in the ```community Cafe repo``` discussion tab.
+> If you can’t find them, please reach out one of the Maintainers below. We're happy to help.
 
-Our Special Interest Groups (SIGs) are:
+<BR>
 
-- Dataplane
-  - [@hoverbear][members-hoverbear] - [@mayadata-io][maya-data] &
-    [@Hoverbear-Consulting](https://github.com/Hoverbear-Consulting)
-  - [@mtzaurus][members-mtzaurus] - [@mayadata-io][maya-data]
-  - [@jonathan-teh][members-jonathan-teh] - [@mayadata-io][maya-data]
-- e2e-testing
-  - [@chriswldenyer][members-chriswldenyer] - [@mayadata-io][maya-data]
-  - [@blaisedias][members-blaisedias] - [@mayadata-io][maya-data]
-- Control Plane
-  - [@tiagolobocastro][members-tiagolobocastro] - [@mayadata-io][maya-data]
-  - [@paulyoong][members-paulyoong] - [@mayadata-io][maya-data]
+### Coding language & style
+> [<img src="https://github.com/openebs/community/blob/develop/images/rust-lang-logo_rust.png" width="100">](https://www.rust-lang.org/learn) <BR>
+> While the <kbd>Replicated PV</kbd> ```MayaStor``` Storage Engine has no formal RFC process, the [Rust RFC template][rust-rfc-template] is an excellent place to derive your issue description from.
 
-Former SIGs (and their members) are:
+| [<img src="https://github.com/openebs/community/blob/develop/images/rustacean-flat-happy.png" width="200">](https://www.rust-lang.org/)  | :crab: The <kbd>Replicated PV</kbd> ```MayaStor``` Storage Engine is ``` 100% ``` developed in the [RUST programming language](https://www.rust-lang.org/)<BR>:crab: We leverage guidance from the RUST community.<BR>:crab: We love Ferris.<BR>:crab: We are #rustaceans |
+| :---         | :---      |
 
-- None, yet!
+---
+### Committing
 
-### Organization FAQs
+You must start you Development work off the `develop` branch. Do **Not use `master`.** - (We do not use that branch name.)
 
-- **What is a _Maintainer_?**
+| [<img src="https://github.com/openebs/community/blob/develop/images/rustacean-flat-gesture.png" width="200">](https://www.rust-lang.org/)  | :star: We run the [bors][bors] ```PR Merge bot tool```.<BR>:star: Bors will merge your commits once they has passed review.<BR>:star: Please ask if you need to [_squash merges_][squash-merges]. We generally do this. It's best to ask first.<BR>:star: Each commit message must adhere to [Conventional Commits][conventional-commits].<BR>:star: You can use [`convco`][tools-convco] if you need a tool to help with commit msgs.<BR>:star: It O.K to force push your branch if you need. Feel free to rewrite commit history it needed. |
```

Comment:
I don't much on markdown syntax but I guess these content in square braces must point to some link, e.g. `bors` , `Conventional Commits` etc

Reply:
Yup. So those links were from the original MayData created doc. 
I've tested them all,  in the original doc and they do point to real websites.... but for some reason the links don't work on the branch when I'm hacking on it. (not sure why).
For example...  [Conventional Commits][conventional-commits] links out to -> https://www.conventionalcommits.org/en/v1.0.0/
- I'm not sure how GitHub converts "[conventional-commits]" to "https://www.conventionalcommits.org/en/v1.0.0/
" but it does. Which is weird, but it works.